### PR TITLE
Adding video to hostnames on Uno "Going Live" page

### PR DESCRIPTION
**Pull request by hjaltedaniel-umbraco**
Adding video to hostname section of Uno docs.

### CODE REVIEW

File: Umbraco-Uno/Going-live/index.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ Check with your DNS host or hostname provider for details on how to configure th
 - Add your hostname to the field and select add
 - To keep track of the process, you can refresh the page when it is done sucessfully it will say: Protected by LATCH  
 
+#### Video tutorial
+
+<iframe width="800" height="450" src="https://www.youtube.com/embed/eOgQKJ1nQbw?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```
Comment (sofietoft) [inline Umbraco-Uno/Going-live/index.md:39]: Update link to embed the newest video on this topic.

```suggestion
<iframe width="800" height="450" src="https://www.youtube.com/embed/4x3DPwQMTr0?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
Great idea adding a video here!

I've suggested updating the link to use the most recent version on the topic.

**Comment by hjaltedaniel-umbraco** [comment]
Link has been updated 👍 
Might be worth also [updating the link on this page](https://our.umbraco.com/documentation/Umbraco-Cloud/Set-Up/Manage-Hostnames/) (that's where I got the old link from 😅 )

**Review by sofietoft — APPROVED** [review]
Ah, thanks for updating the other article as well 💪 

Let's get this merged then!

**Pull request merged**